ArduPlane/quadplane.cpp

```diff
@@ -3074,9 +3076,13 @@ bool QuadPlane::verify_vtol_takeoff(const AP_Mission::Mission_Command &cmd)
  */
 bool QuadPlane::land_detector(uint32_t timeout_ms)
 {
+    // call should_relax() to ensure that lower_limit_start_ms is up to date
+    should_relax();
+
     const uint32_t now = AP_HAL::millis();
     bool might_be_landed =  (landing_detect.lower_limit_start_ms != 0 &&
-                             now - landing_detect.lower_limit_start_ms > 1000);
+                             now - landing_detect.lower_limit_start_ms > 1000 &&
+                             !poscontrol.pilot_correction_active);
```

Comment:
```suggestion
    bool might_be_landed = should_relax() && !poscontrol.pilot_correction_active;
```
I think this is equivalent and slightly clearer, we use the same 1 second threshold in both places.

Reply:
changed, I had to also move the now=millis() call or we'd have a millis race